docs/register-bitcoin-stake.md

```diff
@@ -735,3 +736,24 @@ Rewards can be checked using the `x/incentive` module:
   `babylond query incentive reward-gauges <bech32-address>`
 * **via TypeScript**: You can use the TypeScript implementation to query rewards.
 Please refer to the [TypeScript library documentation](https://github.com/babylonlabs-io/simple-staking/blob/main/src/app/hooks/client/rpc/queries/useBbnQuery.ts).
+
+### 5.3. Costaking Rewards
+
+Costaking enables earning additional rewards by staking both Bitcoin and BABY
+tokens simultaneously.
+
+For costaking one must have both active BTC delegations (via finality providers)
+and BABY token delegations (via validators).
+Costaking rewards are calculated using a user score based on your combined
+stake amounts. The score is based on `min(active_satoshis, active_baby / 
+score_ratio)`.
+
+**Reward flow:**
+1. Fees collected and are accumulated in the costaking reward pool
```

Comment:
Not sure if fees are the best word here, since most of the rewards comes from inflation

A small portion also comes from transaction fees